tailcfg/c2ntypes.go

```diff
@@ -34,6 +34,15 @@ type C2NSSHUsernamesResponse struct {
 	Usernames []string
 }
 
+// C2NUpdateRequest is the request (from control to node) to the /update
+// handler. It tells the node what the latest version is. The node may choose
+// to ignore this update, notify the user, or apply it automatically (for POST
+// requests only).
+type C2NUpdateRequest struct {
+	// LatestVer is the latest released version.
+	LatestVer string
```

Comment:
"Latest" or "LatestVersion". Abbreviating "Ver" here is a bit weird. (in tailcfg.go we already have `IPNVersion`, `OSVersion`, `DistroVersion`, `GoVersion`, `Version`, etc)

Also: what format is this in? Give an example quoted string. The bare "1.50.2" or the full version with all the hash suffixes?

Reply:
Updated, it should be the bare `x.y.z`, which is what we serve from pkgs.